packages/apollo-server-fastify/package.json

```
@@ -26,16 +26,16 @@
     "node": ">=12.0"
   },
   "dependencies": {
-    "@fastify/accepts": "^3.0.0",
-    "@fastify/cors": "^7.0.0",
+    "@fastify/accepts": "^4.0.0",
+    "@fastify/cors": "^8.0.0",
     "apollo-server-core": "file:../apollo-server-core",
     "apollo-server-types": "file:../apollo-server-types"
   },
   "devDependencies": {
     "apollo-server-integration-testsuite": "file:../apollo-server-integration-testsuite"
   },
   "peerDependencies": {
-    "fastify": "^3.17.0",
-    "graphql": "^15.3.0 || ^16.0.0"
+    "fastify": "^3.29.0 || ^4.2.0",
+    "graphql": "^15.8.0 || ^16.5.0"
```

We can't narrow the peer dependency range without a major version bump, so we need to keep the lower bounds as they were:
```suggestion
    "fastify": "^3.17.0 || ^4.2.0",
    "graphql": "^15.3.0 || ^16.5.0"
```

And ideally, we should support a whole major version when we expand (so the 4.2.0 would be 4.0.0 and 16.5.0 would be 16.0.0).